docs-user/gitpod.md

```diff
@@ -0,0 +1,17 @@
+# Setting up profiler on gitpod
+
+Instead of configuring a local setup, you can also use [gitpod](https://www.gitpod.io/), an online continuous development environment  with minimum setup.
+To use gitpod, you can login using your gitHub account and then follow the steps below. 
+
+- Change the URL of your browser to the respository, pull request or issue you want to open on gitpod e.g. for the [profiler project](https://github.com/firefox-devtools/profiler), URL of the upstream repository is `https://github.com/firefox-devtools/profiler`. You can also use the forked repository if you wish to start contributing to the project. 
+- Prefix the URL in the address bar of your browser with `gitpod.io/#` e.g. `https://github.com/firefox-devtools/profiler` becomes `https://gitpod.io/#https://github.com/firefox-devtools/profiler`.
+- Gitpod will then  launch a workspace for you and clone the repository, branch, commit or pull request depending on the URL in the first step.
```

Comment:
These information can be useful, but I would put them below, in a `## Advanced usage` part. Indeed they're for a more advanced usage, I believe most people will want to start with the main profiler page.

In the first part of the file, I think I'd focus on more simple usages, and just reuse the same button we have in the README.